kaizen/cli.py

```
@@ -0,0 +1,136 @@
+        for region, result in results.items():
+            click.echo(f"\nRegion: {region}")
+            click.echo("-" * 30)
+            for test_case in result.get('test_cases', []):
+                status = "✅" if test_case['status'] == 'passed' else "❌"
+                click.echo(f"{status} {test_case['name']}")
+                if 'details' in test_case:
+                    click.echo(f"   Details: {test_case['details']}")
+        
+    except Exception as e:
+        click.echo(f"Error: {str(e)}", err=True)
+        raise click.Abort()
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)